jvm/core/src/main/scala/org/scalatest/concurrent/Eventually.scala

```diff
@@ -438,4 +454,20 @@ trait Eventually extends PatienceConfiguration {
  *   ...
  * </pre>
  */
-object Eventually extends Eventually
+object Eventually extends Eventually {
+
+  //DOTTY-ONLY def callRetry[T](retrying: Retrying[T], timeout: Span, interval: Span, pos: source.Position, fun: => T): T = 
+  //DOTTY-ONLY   retrying.retry(timeout, interval, pos)(fun)
+
+  //DOTTY-ONLY import scala.quoted._
+  //DOTTY-ONLY private[scalatest] def eventuallyMacro[T](timeout: Expr[Span], interval: Expr[Span], fun: Expr[T], retrying: Expr[Retrying[T]])(using quotes: Quotes, typeT: Type[T]): Expr[T] = {
```

Comment:
@liufengyun I am trying to use fun: Expr[=> T] here but unable to, do you mind to help us sanity check if using fun: Expr[T] is the right thing to do?

Thanks.

Reply:
Yes, I think it's correct to use `Expr[T]` here. In macros, `Expr[=>T]` and `Expr[T]` don't have much difference.

Reply:
@liufengyun Thanks for checking!